processor/batchprocessor/README.md

```diff
@@ -29,6 +29,9 @@ regardless of size.
   `0` means no upper limit of the batch size.
   This property ensures that larger batches are split into smaller units.
   It must be greater than or equal to `send_batch_size`.
+- `metadata_keys` (default = empty): When set, this processor will
```

Comment:
I'm missing a lot of context here, but it's unclear to me how "metadata keys" correlates to the OTel data model. Are these resource attribute keys?